templates/understanding-urbit/list.html

```diff
@@ -32,87 +32,79 @@
-    This is a drawing of an Urbit&nbsp;—&nbsp; a new kind of computer. <br /><br />
-    Your Urbit isn’t a physical device: it lives in the cloud or your laptop.
-  </p>
-  <img  class="full c7-10-lg row-1 mt0 mb2 mb4-xl w-100 ba" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/intro/intro-1.svg" alt="A drawing of an Urbit ship." />
-</section>
-<section class="full-bleed full grid12">
-  <p class="full c4-7-lg mt0 mb4 mb2-xl lh-copy">
-    Your Urbit is a tool for you to run your own social software to connect and communicate with your friends and family. <br /><br />
-    Urbit is the opposite of MEGACORP social media.
-  </p>
-  <img class="full c7-10-lg row-1 mt0 mb2 mb4-xl w-100 ba" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/intro/intro-2.svg" alt="An Urbit ship receiving data." />
-</section>
-<section class="full-bleed full grid12">
-  <p class="full c4-7-lg mt0 mb4 mb2-xl lh-copy">
-    Your Urbit is a permanent, private archive.<br /><br />
-    Your Urbit can store all your data and digital assets safely, privately, forever.
-  </p>
-  <img class="full c7-10-lg row-1 mt0 mb2 mb4-xl w-100 ba" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/intro/intro-3.svg" alt="An Urbit ship with its previous versions in a lifetime archive" />
-</section>
-<section class="full-bleed full grid12">
-  <p class="full c4-7-lg mt0 mb6 mb2-xl lh-copy">
-    You own your Urbit like you own Bitcoin.<br /><br />
-    Each Urbit is a piece of a new network that’s owned and controlled by its users.
-  </p>
-  <img class="full c7-10-lg row-1 mt0 mb2 mb7-xl w-100 ba" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/intro/intro-4.svg" alt="An Urbit ship communicating with other ships" />
-</section>
-<section id="megacorp" class="full full-bleed grid12 bg-black">
-  <img class="full c4-10-lg mt7" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/intro/intro-5.jpg"/>
-  <p class="full c4-10-lg white lh-copy mt2 mt4-xl mb7">
-    Today, technology controls us. Every MEGACORP has the same incentive: To keep us hooked on their platform.
-  </p>
-</section>
-<section class="full full-bleed grid12 bg-yellow">
-  <img class="full c4-10-lg mt7" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/intro/intro-6.jpg"/>
-  <p class="full c4-10-lg lh-copy mt2 mt4-xl">
-    We built Urbit so people can control their technology, shape their digital lives and freely build their digital communities.
-  </p>
-  <p class="full c4-10-lg lh-copy mt2">
-    We want to replace the Internet with a foundation for computing that we can all rely on far into the future.
-  </p>
-  <p class="full c4-10-lg lh-copy mt2">
-    Urbit is a large, ambitious project. But at the core it’s simple, durable and yours.
-  </p>
+<section class="c4-10-lg">
+  <h1 id="introduction" class="mb4 full mt4 ">What Urbit Is</h1>
```

Comment:
Just personal taste, but it looks weird and almost internal. "Introduction" is more natural, and all the links say that as well.

Reply:
Agreed.